tests/flow/test_constraint.py

```
@@ -584,6 +584,53 @@ def test08_remove_supporting_index(self):
         drop_node_range_index(self.g, "Author", "nickname")
         drop_node_range_index(self.g, "Author", "birthdate")
 
+    def test09_long_string_values(self):
+        # unique constraints must be enforced regardless of the property
+        # length, the C engine stopped enforcing them past 4096 bytes
+        # (see issue #2440)
+        for length in (4096, 4097, 65536):
+            lbl = f"Doc{length}"
+            hash = 'a' * length
```

Ruff (0.16.1)

[error] Variable `hash` is shadowing a Python builtin

(A001)